charts/internal/shoot-system-components/charts/rook-ceph/templates/toolbox.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: rook-ceph-tools
+  namespace: rook-ceph # namespace:cluster
+  labels:
+    app: rook-ceph-tools
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: rook-ceph-tools
+  template:
+    metadata:
+      labels:
+        app: rook-ceph-tools
+    spec:
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: rook-ceph-tools
```

Comment:
Define some `resources` requirements. Also you might want to define a VPA, see e.g. https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/internal/shoot-system-components/charts/csi-driver-node/templates/vpa.yaml

Reply:
I am going to pass on a VPA for now. This is difficult enough that I don't want to struggle with getting that part of deploying rook/ceph correctly, too. It is an easy addition later.

Resources are different, happy to add.

Reply:
Done